reference/forms/types/repeated.rst

```diff
@@ -177,6 +178,21 @@ type
 The two underlying fields will be of this field type. For example, passing
 ``PasswordType::class`` will render two password fields.
 
+compare
+~~~~~~~
+
+**compare**: ``callable`` **default**: ``null``
+
+The two underlying fields' values will be compared by this callable. The callable receives two values to compare, and should return a boolean: true if the fields are considered equal, false otherwise.
+
+When this option has a null value, it uses internally a callable comparable to:
```

Comment:
You need a second colon here so that the following paragraph is treated as a code block.